### DISCUSSION TITLE
Always Hitting late no matter the Offset

### DISCUSSION THREAD

**Discussion by Tsun1ko** [ppy/osu#30193] [Q&A]
### Type

Game behaviour

### Bug description

So I've been having this
 issue ever since I first tried lazer (like a year ago) that I'm always hitting late no matter the offset and it's really frustrating because lazer runs fine otherwise. It might be a hardware issue because I don't see anyone else talking about this

### Version

2024.1009.1-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17317985/compressed-logs.zip)

**Comment by bdach** [comment]
We're going to need *some* sort of evidence here, video or otherwise. Also maybe information about your setup (computer specs, audio device, input method, monitor...)

**Reply by Tsun1ko** [reply]
my unstable rate looks like this on every map with the same offset and the auto calibrate thingy always tells me that I'm hitting 5-10ms late. I'm already using 5ms offset even though I can play fine with 0ms on stable.

https://github.com/user-attachments/assets/936e868e-d708-4863-bd29-0e00877465b9

CPU: AMD Ryzen 7 2700x
32GB ram
GPU: RTX 2080 SUPER
Audio device: DT 770 pro
input method: ctl472 and drunkdeer A75
montior: Acer Predator XB253Q 240hz

**Reply by Tenexxt** [reply]
can you provide a timing distribution from different scores from different maps?
I don't think that in the play there is a huge offset that makes the gameplay different, especially in a noticable way (exept that one part you fumbled, but come on, that can happen to anyone, anytime and everyone will have a different reason for it happening) and timing distribution from scores instead of clips would be more usefull to determine if something is really wrong rather than having to guess.
Also im asking for scores from different maps so its also easier to determine if you had such problems only on a specific map and its not just a skill issue.

> I'm already using 5ms offset even though I can play fine with 0ms on stable.

The offset settings aren't compatible between stable and lazer as far as i know, so the fact that you can play with 0 offset on stable means nothing (If im right).

**Reply by Tsun1ko** [reply]
sorry for the late reply I didn't play much lazer recently because of tournaments

it's not that bad when playing streams but on jumps it's really noticable
![osu_2024-10-12_21-27-10](https://github.com/user-attachments/assets/535b0516-f602-4bed-9458-b76be0d19561)
![osu_2024-10-14_16-11-28](https://github.com/user-attachments/assets/af2a0e61-8203-4f50-86ff-f32bb9dd6e05)
![osu_2024-10-14_16-16-29](https://github.com/user-attachments/assets/67debe37-1375-4880-bf8b-9c02e3551288)
![osu_2024-10-14_16-23-19](https://github.com/user-attachments/assets/b0f15254-7918-491c-9eb4-5aaa024195e2)

**Comment by peppy** [comment]
This implies either the offset is not working (hard to believe) or something else catastrophic is going on either in your head or hardware. Really going to need something more to work with here, as above.
